### ISSUE TITLE
Hide page in module section

### ISSUE THREAD

**Issue by findthebug** [lukaskleinschmidt/kirby-sortable#48]
I have updated the modules and sortable plugin to the latest version. So far all is working but one thing i can't get done. In the Panel on the right side where the modules are diplayed, also the normal page are shown. how can i disable the normal pages in the sortable section? in the old version of modules and sortable plugin, normal pages are hidden in the module section on the right side. do i miss something? 

![bildschirmfoto 2018-04-17 um 13 34 33](https://user-images.githubusercontent.com/5121132/38867632-d97b3e60-4244-11e8-949a-b6556c16a5ea.png)

**Comment by lukaskleinschmidt** [comment]
How does your blueprint file look like?
Are you using the `modules` or the `sortable` field type?

**Comment by findthebug** [comment]
At the moment im testing it on the projects template with the sortable field type. I did the basic kirby installation and added just the sortable default definition to the blueprint. 

Toolkit Version: 2.5.10
Kirby Version: 2.5.10
Panel Version: 2.5.10

projects.yml
```
title: Projects
files: true
options:
  preview: true
  status: true
  template: true
  url: true
  delete: true  
pages:
  hide: false
  template:
    - module.text
    - module.gallery 
fields:
  title:
    label: Title
    type:  text
  sortable:
    label: Sortable
    type:  sortable
    sortable: true
    layout:  base
    variant: null
    limit: false
    parent: null
    prefix: null
    options:
      limit: false
```

Here is the basic installation to reproduce the behaviour. login/pw root/root
[kirby.zip](https://github.com/lukaskleinschmidt/kirby-sortable/files/1919938/kirby.zip)

**Comment by lukaskleinschmidt** [comment]
Please try to use the `modules` field type. Both fields ship with this plugin.
The `modules` field itself is an extended `sortable` field and [overrides](https://github.com/lukaskleinschmidt/kirby-sortable/blob/master/fields/modules/modules.php) the [`prefix`](https://github.com/lukaskleinschmidt/kirby-sortable#prefix) and [`parent`](https://github.com/lukaskleinschmidt/kirby-sortable#parent) options. This way the plugin knows which pages it should handle.

Your blueprint should look like this.
```yml
title: Projects
files: true
options:
  preview: true
  status: true
  template: true
  url: true
  delete: true  
pages:
  hide: false
  template:
    - module.text
    - module.gallery 
fields:
  title:
    label: Title
    type: text
  modules:
    label: Modules
    type: modules
```

Basically you could also use the `sortable` field if you set the [`prefix`](https://github.com/lukaskleinschmidt/kirby-sortable#prefix) and [`parent`](https://github.com/lukaskleinschmidt/kirby-sortable#parent) option accordingly.

**Comment by findthebug** [comment]
I tried that before but then the modules are displayed on the left side under "Seiten" and on the right side is shown "The parent page could not be found". 

As you can see on the screenshot the title "modules" is on top of "Title". Maybe a Version problem?

![bildschirmfoto 2018-04-17 um 17 13 57](https://user-images.githubusercontent.com/5121132/38879676-3b033a88-4264-11e8-86e5-e61753336ad3.png)

**Comment by lukaskleinschmidt** [comment]
It looks like you are using modules without the modules page wrapper. Did you [configure the modules plugin](https://github.com/getkirby-plugins/modules-plugin#configuration) accordingly?

You would neet to set `c::set('modules.parent.uid', '');` like this to make modules work without the wrapper page.

**Comment by findthebug** [comment]
Okay that was the problem, i missed that option. 
But now the modules itself appears on the left side under the normal pages section. how can i hide it from that list? Thanks for you help!

![bildschirmfoto 2018-04-17 um 18 03 38](https://user-images.githubusercontent.com/5121132/38882260-f51eb9a6-4269-11e8-99b7-d4613780ce8a.png)

**Comment by lukaskleinschmidt** [comment]
You would need to set [`hide: true`](https://getkirby.com/docs/panel/blueprints/page-settings#hide-page) in each module blueprint.

**Comment by findthebug** [comment]
Perfect. Now all is perfect. Thank you for the help and the great plugin!
